cypress/integration/pages/articles/tests.js

```diff
@@ -130,10 +131,9 @@ export const testsThatFollowSmokeTestConfig = ({
         });
       });
 
-      if (serviceHasInlineLink(service) && Cypress.env('APP_ENV') === 'local') {
-        it('should have an inlink link to an article page', () => {
-          cy.get('main a')
-            .eq(1)
```

Comment:
This method only works if the inline link is always the second href in main a. This was not the case across the test assets. In my fix, I have made it so the link selected is the one with the substring '/articles/' in the URL as that will always be an inline link to an article page.

Reply:
We didn't notice this before because 'news' service was passing the test but going to the wrong URL for what we actually want to test.